packages/nodes-base/nodes/Zammad/descriptions/GroupDescription.ts

```diff
@@ -0,0 +1,249 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const groupDescription: INodeProperties[] = [
+	// ----------------------------------
+	//           operations
+	// ----------------------------------
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'group',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a group',
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete a group',
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Retrieve a group',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Get all groups',
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update a group',
+			},
+		],
+		default: 'create',
+	},
+
+	// ----------------------------------
+	//             fields
+	// ----------------------------------
+	{
+		displayName: 'Group Name',
+		name: 'name',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				operation: [
+					'create',
+				],
+				resource: [
+					'group',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'Group Name/ID',
```

Comment:
- Because the entities exist in a fetchable number. Unlike an overly large number like Elasticsearch docs.
- Because the user can easily pick the entity, for better UX.
- Because the guide does not forbid this.